### ISSUE TITLE
Error when trying to switch organizer to counted

### ISSUE THREAD

**Issue by Quinn-D** [neinteractiveliterature/intercode#3230]
Attempted to change signup of an event organizer to counted and got this error.  

![Screenshot_20200203-093157](https://user-images.githubusercontent.com/7717247/73666544-96df9680-4668-11ea-92f9-f81cb62a2cbe.png)

This isn't especially important as the need to do it is questionable, but submitting in case it's helpful.

**Comment by dkapell** [comment]
This is because the signup record for 'team_member' doesn't have a bucket,
as it's not a bucketed signup.  The reasonable thing here is probably to
disallow team members from being able to enter this dialog for team members.

**Comment by nbudin** [comment]
@dkapell Yeah, agreed.  Let's stop people from doing a thing that makes no sense to try to do.

**Comment by Quinn-D** [comment]
While I agree it is unlikely one would want to do it, I think there are reasonable reasons why one might.  In this situation it was to have organizers show up in attendee count for an unlimited signup social event.
